### PR TITLE
Fix: performance issue related to a list usage.

### PR DESCRIPTION
**Pull request by hadialqattan**
Fixes: #140

### CODE REVIEW

File: unimport/constants.py [hunk 1]
```diff
@@ -6,9 +6,8 @@
 VERSION = "0.3.0"
 
 PY38_PLUS = sys.version_info >= (3, 8)
-PY37_PLUS = sys.version_info >= (3, 7)
 
-SUBSCRIPT_TYPE_VARIABLE = [
+SUBSCRIPT_TYPE_VARIABLE = {
```
Comment (hakancelik96) [inline unimport/constants.py:10]: Hi, thank you, Here we can make it more efficient using frozenset.

```suggestion
SUBSCRIPT_TYPE_VARIABLE = frozenset({
```

Comment (hadialqattan) [reply unimport/constants.py:10]: Yeah, you're right!

Comment (hadialqattan) [reply unimport/constants.py]: I've changed it to `frozenset`, but to be honest, there's no difference between the normal `set` efficiency and the `frozenset`!

Comment (hadialqattan) [reply unimport/constants.py]: My pleasure... you're right, changed!

**Pull request merged**